### PR TITLE
GH-86: Notification in channel on MR reopen

### LINKED ISSUE #86: Re-opening a MR doesn't cause post a message

### PR DESCRIPTION
**Pull request by ashishbhate**
Send notification in channel on MR reopen. Fixes #86 

I had to update my go.mod file to get the plugin to work. I'm not sure if the CI needs it. 

See messages starting from https://community.mattermost.com/core/pl/nj3bdn8b8ifjfp9r8w41frk84w

### CODE REVIEW

**Comment by ashishbhate** [comment]
CI is failing with the same error I was seeing with the dependency versions in the `go.mod` file. There's a link in the OP to the community server chat log where I've posted how I fixed this error.

**Comment by levb** [comment]
The build is failing because of `go.mod` and go 1.13, fix should be coming to master shortly.

File: server/webhook/merge_request.go [hunk 1]
```diff
@@ -73,6 +73,8 @@ func (w *webhook) handleChannelMergeRequest(event *gitlab.MergeEvent) ([]*Handle
 		message = fmt.Sprintf("[%s] Merge request [#%v %s](%s) was merged by [%s](%s)", repo.PathWithNamespace, pr.IID, pr.Title, pr.URL, senderGitlabUsername, w.gitlabRetreiver.GetUserURL(senderGitlabUsername))
 	} else if pr.Action == "close" {
 		message = fmt.Sprintf("[%s] Merge request [#%v %s](%s) was closed by [%s](%s)", repo.PathWithNamespace, pr.IID, pr.Title, pr.URL, senderGitlabUsername, w.gitlabRetreiver.GetUserURL(senderGitlabUsername))
+	} else if pr.Action == "reopen" {
```
Comment (cpoile) [inline server/webhook/merge_request.go]: Could we make this a bit cleaner with a switch statement?

Comment (ashishbhate) [reply server/webhook/merge_request.go]: Sounds good to me. I thought doing this in the first commit, but I wasn't sure if the larger diff for a small change like this was a good idea.

**Comment by ashishbhate** [comment]
Updated with switch instead of long if-else. Let me know if you need me to squash the two commits into a single commit.

**Comment by levb** [comment]
@ashishbhate can you please sync and merge with master to fix the build?

**Comment by ashishbhate** [comment]
@levb I've merged master into the branch

**Comment by jasonblais** [comment]
@DHaussermann Friendly reminder to help with QA review :)

**Comment by DHaussermann** [comment]
@levb this PR is still untested as I unable to build this. Let's follow-up so I can get assistance from someone on the team to see what's going on.

**Comment by ashishbhate** [comment]
Hi guys, let me know if there's anything I can do to help get this PR merged.

**Pull request merged**